deploy/yamls/zfsbackup-crd.yaml

```diff
@@ -0,0 +1,108 @@
+
+
+##############################################
+###########                       ############
+###########   ZFSBackup CRD       ############
+###########                       ############
+##############################################
+
+# ZFSBackups CRD is autogenerated via `make manifests` command.
+# Do the modification in the code and run the `make manifests` command
+# to generate the CRD definition
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.8
+  creationTimestamp: null
+  name: zfsbackups.zfs.openebs.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.prevSnapName
+    description: Previous snapshot for backup
+    name: PrevSnap
+    type: string
+  - JSONPath: .status
+    description: Backup status
+    name: Status
+    type: string
+  - JSONPath: .metadata.creationTimestamp
+    description: Age of the volume
+    name: Age
+    type: date
+  group: zfs.openebs.io
+  names:
+    kind: ZFSBackup
+    listKind: ZFSBackupList
+    plural: zfsbackups
+    shortNames:
+    - zb
+    singular: zfsbackup
+  preserveUnknownFields: false
+  scope: Namespaced
+  subresources: {}
+  validation:
+    openAPIV3Schema:
+      description: ZFSBackup describes a zfs backup resource created as a custom resource
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ZFSBackupSpec is the spec for a ZFSBackup resource
+          properties:
+            backupDest:
+              description: BackupDest is the remote address for backup transfer
+              minLength: 1
+              type: string
+            ownerNodeID:
+              description: OwnerNodeID is a name of the nodes where the source volume
+                is
+              minLength: 1
+              type: string
+            prevSnapName:
```

Comment:
q_: Are there names in sync with the proposed refactoring changes planned in velero plugin?

Reply:
>proposed refactoring changes planned in velero plugin
not aware of this? can you share the proposal link.